### ISSUE TITLE
[A11y] Address A11y issues in Settings -> Notebook Images

### ISSUE THREAD

**Issue by jenny-s51** [opendatahub-io/odh-dashboard#1100]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Table header text should not be empty:
- Notebook image settings - list
- Notebook image settings - import new image modal

Axe/VO issues of varying severities throughout screens/modals in:
- Notebook image settings - list
- Notebook image settings - import new image modal

### Expected Behavior

Fix a11y violations and test with VO to resolve issues.
